index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Interesting.  You shouldn't need to use `gulp-haser` -- the `gulp-cache-breaker` plugin ads a `SHA` to the filename for you.  For instance in the end it should read:

```
<link href=css/toastr.abd123.css" />
```

The path provided should be a file system path, since we read in the file and make a SHA of it's contents.   That said, once it writes out to HTML it should be a url!  On linux the path separator for a file path is the same as that for a url -- in window this isn't the case, hence the bug!

I think your patch is likely nearly there, but I think there's a few other things we might need to change.   I'll pull down the branch in the near term and make sure things are working as expected.

Thanks!